virttest/data_dir.py

```diff
@@ -214,8 +216,16 @@ def get_tmp_dir(public=True):
 
 
 def get_download_dir():
-    if not os.path.isdir(DOWNLOAD_DIR):
-        shutil.copytree(BASE_DOWNLOAD_DIR, DOWNLOAD_DIR)
+    """
+    Returns the dir that holds download definition files, AKA asset files
+
+    Before retuning the download directory, it makes sure the user, writable
+    copy of the download dir is created, and that all asset definition
+    file are in sync.
+
+    :returns: a writable directory populated with asset definition files
+    """
+    dir_util.copy_tree(BASE_DOWNLOAD_DIR, DOWNLOAD_DIR)
```

Comment:
I'm wondering why this function is even there? The files are downloaded into `destination` location, which does not even uses the `downloaddir` as base.

I replaced this function with `return BASE_DOWNLOAD_DIR`, made the BASE_DOWNLOAD_DIR read only and everything worked well (the JeOS was read from the BASE_DOWNLOAD_DIR/jeos-23-64.ini and downloaded to avocado_data_dir).